src/webview.ts

```diff
@@ -0,0 +1,60 @@
+import sys from "./ffi.ts";
+
+function encode(value: string): Deno.UnsafePointer {
+  const encoded = new TextEncoder().encode(value);
+  const buffer = new Uint8Array(encoded.byteLength + 1);
+  buffer.set(encoded, 0);
+
+  buffer[buffer.byteLength - 1] = 0x00;
+  return Deno.UnsafePointer.of(buffer);
+}
+
+export type SizeHint = 0 | 1 | 2 | 3;
+
+export const SizeHint = {
+  NONE: 0,
+  MIN: 1,
+  MAX: 2,
+  FIXED: 3,
+} as const;
+
+export class Webview {
+  #handle: Deno.UnsafePointer | null;
+  #url?: string;
```

Comment:
Should these be marked `private`? What is the meaning of the `#` here?

Reply:
It's the native sugar to make a class field private at runtime (with an associated computation cost at runtime)
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/Private_class_fields

Reply:
yes, these are truly private (private at runtime) fields.

Reply:
![giphy](https://user-images.githubusercontent.com/10974/148780494-f59b2c7b-7682-41c5-873e-eb3f693d8ab1.gif)